### PR TITLE
Replace broken wiki short URLs with full URLs

### PR DESCRIPTION
**Pull request by daniel-beck**
Short URLs were nice to save bytes on `update-center.json`, but they no longer work.

### CODE REVIEW

File: resources/artifact-ignores.properties [hunk 1]
```diff
@@ -219,29 +219,29 @@ sonargraph-plugin = https://github.com/jenkins-infra/update-center2/pull/405
 sonatype-ci = https://wiki.jenkins.io/x/iYDPAw
```
Comment (daniel-beck) [inline resources/artifact-ignores.properties:219]: https://wiki.jenkins.io/display/JENKINS/63930505 perhaps?

Comment (MarkEWaite) [reply resources/artifact-ignores.properties:219]: +1 from me to use that replacement.

**Review by MarkEWaite — APPROVED** [review]
Thanks for fixing the broken links!

**Review by dduportal — APPROVED** [review]

**Comment by dduportal** [comment]
@daniel-beck May I let you merge if ready to go?

**Pull request merged**